airdrop-claim-reject-cancel-hts-token-js-sdk/utils/accountCreate.js

```diff
@@ -0,0 +1,12 @@
+import { AccountCreateTransaction } from "@hashgraph/sdk";
```

Comment:
consider combining all 3 of the util files into a single file.

Reply:
N/A. Keeping separate to delineate by service/functionality/purpose.